verticapy/connection/read.py

```diff
@@ -63,6 +70,41 @@ def read_dsn(section: str, dsn: Optional[str] = None) -> dict:
     -------
     dict
         dictionary with all the credentials.
+
+    Example
+    -------
```

Comment:
Nice example, please add lines to explain what you do (better than comments).
Please look at what your output looks like (do you have a pic to share so I understand how it renders the result?)

Reply:
![image](https://github.com/vertica/VerticaPy/assets/40260999/72fcadc3-479e-48f3-a684-9287fb16710d)